doc/install.rst

```diff
@@ -28,21 +28,26 @@ Ubuntu/Mint
 .. index:: Ubuntu Linux
 .. index:: Mint
 
-We maintain a `Ubuntu Repository <https://gerbera.jfrog.io/>`__.
+We maintain a `Ubuntu Repository <https://pkg.gerbera.io/>`__.
```

Comment:
I'm getting this friendly message for the page:
![grafik](https://github.com/gerbera/gerbera/assets/23654788/ad2b701f-d200-4bcb-b151-1431c30f579f)

Reply:
Yes a landing page there would be nice to have. Some sources don't though and makes finding if a release is available by directory read traversal a little harder to determine if a certain distro is supported yet.